docs/advanced-topics/dockstore-cli/dockstore-cli-faq.rst

```diff
@@ -103,7 +113,23 @@ If a Docker lockup happens, you will notice in-progress WDL tasks do not progres
 
 The other issue we often see is some instances of scattered tasks getting `sigkilled <https://www.gnu.org/software/libc/manual/html_node/Termination-Signals.html>`__ by the operating system. You will know when this happens because the `rc` (return code) file will read 137. If it reads anything except 137, then you can assume that it wasn't actually a resource management error and look in stderr or stdout for the true culprit. For more on return codes, see :ref:`this FAQ <return-code-wdl>` entry.
 
-To prevent these issues from happening, we recommend setting up your Cromwell configuration file to limit how many scattered tasks run at once, and then setting up the Dockstore CLI to make use of that Cromwell configuration file. :doc:`A step-by-step tutorial is available here. </advanced-topics/dockstore-cli/local-cromwell-config>`
+To prevent these issues from happening, we recommend setting up your Cromwell configuration file to limit how many scattered tasks run at once, and then setting up the Dockstore CLI to make use of that Cromwell configuration file. :doc:`A step-by-step tutorial is available here. </advanced-topics/dockstore-cli/local-cromwell-config>` 
+
+:ref:`(back to top) <topCLIFAQ>`
+
+What environment do you test tools in?
+--------------------------------------
+
+Typically, we test running tools in Ubuntu Linux 16.04 LTS on VMs in
```

Comment:
I feel like this came from PCAWG originally, we don't really regularly test workflows on a standardized machine anymore, so this section could probably just be cut.

Reply:
e.g. our tooltester platform for metrics is now built on AWS AGC where each workflows presumably requests its own job sizes independently from AWS batch

Reply:
Removed the question entirely and replaced it with a question about launching workflows in general in the general FAQ